server/api/api.go

```diff
@@ -29,6 +29,9 @@ type Subscriptions interface {
 
 type Calendar interface {
 	ViewCalendar(from, to time.Time) ([]*remote.Event, error)
+	CreateEvent(event *remote.Event) (*remote.Event, error)
+	CreateCalendar(calendarName string) (*remote.Calendar, error)
```

Comment:
1/5 make it `CreateCalendar(calendar *remote.Calendar)` - may seem redundant now but (a) consistent with others, (b) chances are will want to do it eventually anyway, for more parameters.

Reply:
Agree and will make the adjustment.